rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/LicenseSpecTest.java

```diff
@@ -213,4 +214,13 @@ public void should_document_delete_all_license_info() throws Exception {
                 .andExpect(status().isOk());
     }
 
+    @Test
+    public void should_document_import_osadl_info() throws Exception {
+        String accessToken = TestHelper.getAccessToken(mockMvc, testUserId, testUserPassword);
+        mockMvc.perform(post("/api/licenses/" + "/import/OSADL")
```

Comment:
`"/api/licenses/" + "/import/OSADL" `can be replaced by `"/api/licenses/import/OSADL"`